plugins/disk/du-2

```diff
@@ -160,15 +161,21 @@ sub cache_is_too_old {
 
 sub du_not_running {
     if (-e $LOCKFILE) {
-        my ($time) = `cat $TIMEFILE`;
-        chomp $time;
-        if ( (time - $time) > ($ENV{interval}*60*60) ) {
-            # The cache is really old (60xinterval) => Maybe the lockfile wasn't properly deleted.
-            # Let's delete it.
-            system("rm $LOCKFILE;");
-            return 1;
+        if (-e $TIMEFILE) {
+            my ($time) = `cat $TIMEFILE`;
+            chomp $time;
+            if ( (time - $time) > ($ENV{interval}*60*60) ) {
+                # The cache is really old (60xinterval) => Maybe the lockfile wasn't properly deleted.
+                # Let's delete it.
+                system("rm $LOCKFILE;");
```

Comment:
Maybe you could improve this old code by using `unlink($LOCKFILE);` here :)

Reply:
i updated the commit as suggested.